res/xml/bone.xml

```diff
@@ -0,0 +1,45 @@
+<?xml	version="1.0"	encoding="utf-8"?>
+<keyboard>
+	<row>
+		<key   width="1"   key0="j"							key2="°"						key4="esc"/>
+		<key   width="1"   key0="d"							key2="§"						key4="_"/>
+		<key   width="1"   key0="u"							key2="ℓ"						key4="["/>
+		<key   width="1"   key0="a"							key2="»"						key4="]"/>
+		<key   width="1"   key0="x"							key2="«"						key4="^"/>
+		<key   width="1"   key0="p"			key1="$"						key3="!"		key4="7"/>
+		<key   width="1"   key0="h"			key1="€"		key2="⟨"		key3="&lt;"		key4="8"/>
+		<key   width="1"   key0="l"			key1="£"		key2="⟩"		key3="&rt;"		key4="9"/>
+		<key   width="1"   key0="m"			key1="¥"						key3="="		key4="f11_placeholder"/>
+		<key   width="1"   key0="w"			key1="¢"						key3="&amp;"	key4="f12_placeholder"/>
+	</row>
+	<row>
+		<key   width="1"   key0="c"							key2="tab"						key4="\"/>
+		<key   width="1"   key0="t"			key1="accent_circonflexe"	key2="accent_caron"	key4="/"/>
```

Comment:
The dead keys are visually heavy and more and more will be added to the keyboard.
They will be added automatically to the layout depending on the user's installed languages so you can remove them all.
If you wish to dedicate a place for some of them but to keep them hidden by default, you can add the `loc ` prefix.

Reply:
This is kinda intentional because I think it is part of the Bone Keyboard Philosophy to be able to type a lot of characters. But I guess it wouldn't hurt to make them hide according to the user's installed languages, because everyone can still enable to show all in the settings.

Reply:
ill do this later